docs/lib/datastore/fragments/ios/getting-started/30_platformIntegration.md

```diff
@@ -1,31 +1,20 @@
-The `Amplify/Tools` pod provides an integration layer between the Amplify CLI headless mode (i.e. no AWS credentials needed) and Xcode. It integrates into the build process of the IDE to provide a seamless experience.
+The `amplify-app` library provides an integration layer between the Amplify CLI headless mode (i.e. no AWS credentials needed) and Xcode.
```

Comment:
I wouldn't call this a "library". I'd say the `npx amplify-app` command.

Reply:
maybe _utility_?